doc/classes/GraphElement.xml

```diff
@@ -66,4 +66,8 @@
 			</description>
 		</signal>
 	</signals>
+	<theme_items>
+		<theme_item name="resizer" data_type="icon" type="Texture2D">
```

Comment:
This one has description in GraphNode.

Reply:
Wait, that's weird. It should've been removed from `GraphNode`.

Reply:
Ah! I think there are going to be a few more duplicated items to remove.